lib/features/popup-menu/PopupMenuComponent.js

```diff
@@ -246,7 +249,7 @@ export default function PopupMenuComponent(props) {
           />
         </div>
         ${ entries.length === 0 && html`
-          <div class="djs-popup-no-results">No matching entries found.</div>
+          <div class="djs-popup-no-results">${ noSearchResultsEntry || 'No matching entries found.' }</div>
```

Comment:
Idea: Provide the default in `PopupEditor` so we can translate the string, too.

Reply:
Can you explain? 🤔

Reply:
Other idea: Consider if we want to compute the result dynamically:

```javascript
<div class="djs-popup-no-results">${ noResultsEntry(value) }</div>
```

Reply:
Consider I want to add an awesome box:

```html
We did not find an entry matching `<foo>`, do you want to <a href="https://some-market-palce-api?search=${foo}`">Market place</a>
```

Reply:
I was wondering if something like that would be useful. Will try that out.